iphone/Classes/TiAppiOSSearchableItemProxy.h

```diff
@@ -5,10 +5,10 @@
  * Please see the LICENSE included with this distribution for details.
  */
 #if IS_XCODE_7
+#ifdef USE_TI_APPIOS
```

Comment:
The legacy plan was to not replace every constant as part of this PR, but yes it makes sense. Will update the remaining constants. Thanks!